.github/scripts/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM fedora:34
+RUN dnf -y install glibc cyrus-sasl-lib cyrus-sasl-plain cyrus-sasl-gssapi cyrus-sasl-md5 libuuid openssl gettext hostname iputils libwebsockets-devel libnghttp2 gdb valgrind && dnf -y update && dnf clean all
+ADD qpid-proton-image.tar.gz skupper-router-image.tar.gz /
```

Comment:
The build strategy should mirror production usage. If the production build is going to compile Proton as a static library to be linked into Skupper Router, then this should probably be followed (and it also should be followed in the Skupper Router CI, which it currently is not).

@mcressman afaik the plan is to continue to compile static Proton into the qdrouterd, right?

Reply:
Yes, we will continue static compile downstream.  Downstream is useful because downstream product releases for clients/proton might be only twice a year and we may need to use a more recent version.  Upstream is optional, but as you suggest, doing the same upstream and downstream is better.  I'd say do the static compile upstream for now and see if we run into any issues.

Reply:
@mcressman Thanks for the info! Would it be possible to opensource the .spec file that you maintain (and put a copy of it to this repo, so it can be directly used to do builds?)